packages/lesswrong/lib/collections/revisions/resolvers.js

```diff
@@ -110,7 +106,12 @@ addFieldsDict(Revisions, {
     resolveAs: {
       type: 'String',
       resolver: ({html}) => {
-        const mainTextHtml = sanitizeHtml(html, { nonTextTags: nonMainTextTags })
+        const mainTextHtml = sanitizeHtml(
+          html, { 
+            allowedTags: _.without(sanitizeHtml.defaults.allowedTags, 'blockquote'),
+            nonTextTags: ['blockquote']
```

Comment:
I think this is going to let through `img` tags, where it didn't before, which would be bad.

Reply:
It actually doesn't let through image tags, where before it did.

Reply:
(this is because image tags are not included in the default.allowedTags. Although we'd need to add them in manually if we switch to using the export)

Reply:
Ah, sorry I'm dumb. But, fixed now.